pydata/models.py

```diff
@@ -0,0 +1,22 @@
+import re
+
+from workshops.models import Person as BasePerson
+from workshops.models import Task as BaseTask
+
+
+class Person(BasePerson):
+    """ Represents a PyData speaker."""
+
+    PROFILE_REGEX = re.compile(r'^(?P<url>.+?(?=/speaker))/speaker/profile/(?P<id>[^/]+)/?')
+
+    class Meta:
```

Comment:
Since these models are just used to override behaviour (if any), they should not exist in the database.
Without `abstract=True`, the django test framework complains.